web/themes/custom/cloudy2/src/sass/layout/_page.content.scss

```diff
@@ -3,3 +3,6 @@
 .page__content {
 
 }
+.homepage-grid .views-row .views-col:first-child {
```

Comment:
Hmm. I feel like the grid should have something for this. I'm approving this for now, but making a mental note to come back and figure out what grid class we can add to the view to achieve the same thing. 🤔